File: source/includes/connect/stable-api.scala
import com.mongodb.{ServerApi, ServerApiVersion}
import org.mongodb.scala._

object StableAPI {

  def main(args: Array[String]): Unit = {

    {
        // start-enable-stable-api
        val serverApi = ServerApi.builder()
            .version(ServerApiVersion.V1)
            .build()

        // Replace the uri string placeholder with your MongoDB deployment's connection string
        val uri = "<connection string URI>"

        val settings = MongoClientSettings.builder()
            .applyConnectionString(ConnectionString(uri))
            .serverApi(serverApi)
            .build()

        val mongoClient = MongoClient(settings)
        // end-enable-stable-api
    }

    {
        // start-stable-api-options
        val serverApi = ServerApi.builder()
            .version(ServerApiVersion.V1)
            .strict(true)
            .deprecationErrors(true)
            .build()
        // end-stable-api-options
    }

    mongoClient.close()
  }
}

File: source/includes/connect/stable-api.php
<?php

// start-specify-v1
$uri = 'mongodb://<hostname>:<port>';

$driverOptions = ['serverApi' => new MongoDB\Driver\ServerApi('1')];

$client = new MongoDB\Client($uri, [], $driverOptions);
// end-specify-v1

// start-stable-api-options
$uri = 'mongodb://<hostname>:<port>';

$serverApi = new MongoDB\Driver\ServerApi('1', strict: true, deprecationErrors: true);
$driverOptions = ['serverApi' => $serverApi];

$client = new MongoDB\Client($uri, [], $driverOptions);
// end-stable-api-options
